### PR TITLE
DT-99-fix-homepage-on-mobile

### PR DESCRIPTION
**Pull request by Giang-st1-00**
handle event in homepage on mobile device

### CODE REVIEW

File: components/common/Video/NormalVideo/index.tsx [hunk 1]
```diff
@@ -122,6 +122,18 @@ export default function NormalVideo({ data }: INormalVideo) {
                     ></div>
                 )}
 
+                <div
+                    onClick={() => {
```
Comment (Duy-Tinh-Dev) [inline components/common/Video/NormalVideo/index.tsx:126]: create function

File: components/common/Video/DRMVideo/index.tsx [hunk 2]
```diff
@@ -8,11 +8,11 @@ import ControlsVideo from "../ControlsVideo";
 import { IDRMVideoShortPayload } from "@/model/video";
 import Comments from "../../Comments/Comments";
 
-interface IVideo {
+interface IDRMVideo {
```
Comment (Duy-Tinh-Dev) [inline components/common/Video/DRMVideo/index.tsx]: IDRMVideoProps

File: components/common/Video/ActionVideo/index.tsx [hunk 3]
```diff
@@ -25,13 +25,13 @@ export default function ActionVideo({
...
     return (
-        <div className="actionVideo opacity-0 transition duration-500 ease-in-out absolute top-[18%] sm:top-[25%] right-6 flex flex-col">
+        <div className="actionVideo opacity-0 transition duration-500 ease-in-out absolute top-[18%] sm:top-[25%] right-6 flex flex-col scale-90 sm:scale-100">
             <div className="mb-5 flex justify-center relative">
                 <div className="w-[40px] h-[40px] rounded-[50%] overflow-hidden flex justify-center bg-[#b7b7b7]">
                     <Image width={40} height={40} alt="" src={pathAvatar} />
                 </div>
                 <div>
-                    <TiPlus fontSize={20} className="absolute top-[-5px] right-[-7px]" onClick={() => {}} />
+                    <TiPlus className="text-[20px] absolute top-[-5px] right-[-7px]" onClick={() => {}} />
```
Comment (Duy-Tinh-Dev) [inline components/common/Video/ActionVideo/index.tsx]: delele onClick if not using

File: components/common/Video/DRMVideo/index.tsx [hunk 4]
```diff
@@ -8,11 +8,11 @@ import ControlsVideo from "../ControlsVideo";
 import { IDRMVideoShortPayload } from "@/model/video";
 import Comments from "../../Comments/Comments";
```
Comment (Duy-Tinh-Dev) [inline components/common/Video/DRMVideo/index.tsx]: @

Comment (Giang-st1-00) [reply components/common/Video/DRMVideo/index.tsx]: @DuyTinh2003  
image

**Pull request merged**